tests/spread/core24/python-hello/uv/src/pyproject.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "example"
```

It may be worth setting the name to something more meaningful
```suggestion
name = "hello"
```